_posts/2024-05-14-switch-to-bam.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: What does it take to switch over to bam from fastq?
+tags:
+  - bam
+  - fastq 
+contributors:
+  - lskatz
+  - SeanSierra-Patev
+reviewers:
+  - pmenzel
+---
+
+It has been over 14 years since the formalization of the fastq format {% cite Cock2009 %}.
+It describes sequences and their quality scores.
```

Comment:
```suggestion
It has been over 14 years since the formalization of the fastq format {% cite Cock2009 %},
which describes sequences and their quality scores.
```

not sure L15 needs to be its own sentence